docker/build.sh

```diff
@@ -24,11 +35,18 @@ usage() {
 Usage: $(basename ${BASH_SOURCE[0]}) [<options>] <service>...
 
 Options:
-  -D, --docker-repo  Specify a Docker repository
+  -D                 Specify a Docker repository
                      (default: unspecified, i.e., system default)
-  -t, --tag          Tag the Docker image (default: $DOCKER_TAG)
-  --ask-push         Offer to push newly built images to registry
-  --no-cache         Pass --no-cache to docker-build
+  -t                 Tag the Docker image (default: $DOCKER_TAG)
+  -P                 Offer to push newly built images to registry
+  -C                 Pass --no-cache to docker-build
+
+Notes:
+  <service> defaults to 'server client'.
+
+  You can build all services with 'all'. Currently supported are:
+    client server media-service pgbouncer postfix repmgr
```

Comment:
Please dynamically build this line by using `TARGETS`, so one does not forget to update this place if some targets are changed.